homeassistant/components/abode/binary_sensor.py

```diff
@@ -38,4 +41,6 @@ def is_on(self):
     @property
     def device_class(self):
         """Return the class of the binary sensor."""
+        if self._device.get_value("is_window"):
```

Comment:
```suggestion
        if self._device.get_value("is_window") == "1":
```

Reply:
As long as its never going to be "0" it should be fine.

Reply:
Hm, it looks like that's the issue. Some how when `is_window` is an empty string, it's actually returning the string `0` from Abodepy. Here's a quick test I did - printing the results of:

```
bool(self._device.get_value("is_window"))
self._device.get_value("is_window")
```

Garage Entry Door: True - 0
Bedroom Window: True - 1

Looking into the abodepy side to see why this is.